integration-tests/hibernate-orm-panache-kotlin/src/main/kotlin/io/quarkus/it/panache/kotlin/TestEndpoint.kt

```diff
@@ -1018,4 +1018,45 @@ class TestEndpoint {
         Assertions.assertNull(f.getAnnotation(XmlAttribute::class.java))
         Assertions.assertNotNull(f.getAnnotation(XmlTransient::class.java))
     }
+
+
+    @GET
+    @Path("9036")
+    @Transactional
+    fun testBug9036(): String {
```

Comment:
This is not a bug but a feature ;)
Maybe `testIssue9036` or `testFilter` ?

Reply:
Who cares, really? The issue number is what matters.

Reply:
Me ? But OK it's nitpicking ;)
I don't care about issue number for what is not a bug.
Let's keep it if you want.